pkgs/tools/networking/wait-for-it/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

Comment:
Shouldn't this be `stdenvNoCC.mkDerivation` ?

Reply:
I'm unfamiliar with this command and didn't see it in the Nix Pills or contributing packages guide. So I'm in no position to debate it. Do you have any documentation that could help here?

Reply:
Sadly, I don't know if there is documentation for `stdenvNoCC`. It's basically `stdenv` but without an implicit dependency on `gcc`. This should reduce the amount of build dependencies needed to build this package from scratch.

Reply:
Thanks. Done. I guess I'm just wondering how did *you* originally find out about stdenvNoCC? Even with no docs. Would love to myself be able to contribute and let the next person just know this.